### PR TITLE
cpan: add page

### PR DESCRIPTION
**Pull request by shxntanu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/cpan.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cpan
+
+> Query, download and build perl modules from CPAN sites.
+> More information: <https://manned.org/man/cpan>.
+
+- Install modules (sole -i is optional):
```
Comment (spageektti) [inline pages/common/cpan.md:6]:
```suggestion
- Install a module (sole -i is optional):
```

File: pages/common/cpan.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`cpan -i {{module_name}}`
+
+- Force install modules (must use -i):
```
Comment (spageektti) [inline pages/common/cpan.md:10]:
```suggestion
- Force install a module (must use -i):
```

**Comment by spageektti** [comment]
Hi, thanks for your contribution

File: pages/common/cpan.md (hunk 2 above)
Comment (shxntanu) [reply pages/common/cpan.md]: Done, applied both suggestions.

File: pages/common/cpan.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# cpan
+
+> Query, download and build perl modules from CPAN sites.
+> More information: <https://manned.org/man/cpan>.
```
Comment (spageektti) [inline pages/common/cpan.md:4]:
```suggestion
> More information: <https://manned.org/cpan>.
```

Comment (shxntanu) [reply pages/common/cpan.md]: I've incorporated the suggestion, but both URLs lead to the same page.

Comment (spageektti) [reply pages/common/cpan.md]: > I've incorporated the suggestion, but both URLs lead to the same page.

I know, but we use `manned.org` without `/man/` on all other pages, and it is shorter.

File: pages/common/cpan.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# cpan
+
+> Query, download and build perl modules from CPAN sites.
+> More information: <https://manned.org/cpan>.
+
+- Install a module (sole -i is optional):
+
+`cpan -i {{module_name}}`
```
Comment (spageektti) [inline pages/common/cpan.md:8]:
```suggestion
`cpan {{-i}} {{module_name}}`
```

File: pages/common/cpan.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# cpan
+
+> Query, download and build perl modules from CPAN sites.
+> More information: <https://manned.org/cpan>.
+
+- Install a module (sole -i is optional):
```
Comment (spageektti) [inline pages/common/cpan.md:6]:
```suggestion
- Install a module (`-i` is optional):
```

File: pages/common/cpan.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`cpan -i {{module_name}}`
+
+- Force install a module (must use -i):
```
Comment (spageektti) [inline pages/common/cpan.md:10]:
```suggestion
- Force install a module (`-i` is not optional):
```

**Comment by shxntanu** [comment]
Applied all suggested changes. Thank you for reviewing!! @sebastiaanspeck @spageektti

**Review by spageektti — APPROVED** [review]

**Pull request merged**